lighty-modules/lighty-openflow-sb/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.lighty.core</groupId>
+        <artifactId>lighty-parent</artifactId>
+        <version>9.2.1-SNAPSHOT</version>
+        <relativePath>../../lighty-core/lighty-parent</relativePath>
+    </parent>
+
+    <groupId>io.lighty.modules</groupId>
+    <artifactId>lighty-openflow-sb</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.lighty.core</groupId>
+            <artifactId>lighty-controller</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.opendaylight.openflowplugin</groupId>
+            <artifactId>openflowplugin-api</artifactId>
+            <version>0.7.1</version>
```

Comment:
please make entry in https://github.com/PantheonTechnologies/lighty-core/blob/9.2.x/lighty-core/dependency-versions/pom.xml for all OF artefacts with version.